shell/components/nav/Group.vue

```diff
@@ -302,6 +303,7 @@ export default {
       }
       > H6 {
         text-transform: none;
+        padding: 8px 0px 8px 16px;
```

Comment:
A bit of a nitpick: the uom is redundant with a 0 value

```suggestion
        padding: 8px 0 8px 16px;
```